torch/nn/modules/batchnorm.py

```diff
@@ -93,7 +110,8 @@ class BatchNorm1d(_BatchNorm):
         eps: a value added to the denominator for numerical stability.
             Default: 1e-5
         momentum: the value used for the running_mean and running_var
-            computation. Default: 0.1
+            computation. Can be set to ``None`` for cumulative moving average
+            ("simple" average). Default: 0.1
```

Comment:
these are indeed simple average, maybe we can remove the quotes? or we can call it arithmetic average?